src/shell.ts

```diff
@@ -20,13 +20,13 @@ import { ChildProcess } from 'child_process';
 import * as shelljs from 'shelljs';
 
 const WINDOWS = 'win32';
-const MACOS  = 'darwin';
-const LINUX  = 'linux';
+const MACOS = 'darwin';
+const LINUX = 'linux';
 
 export enum Platform {
-    WINDOWS,
-    MACOS,
-    LINUX
+    WINDOWS = 1,
```

Comment:
Wondering if it would be nice to add NONE or INVALID with string '' and num value 0.

Pros:
Clearly shows the intent that number 0 shouldn't be used and is an invalid value
Would fail for the cases where it should be failing (when checking if platform exists, undefined, null, and the NONE value (either empty string or 0 depending on using the const or the enum value) would all return false.

Cons:
False positives when checking for undefined variables.

Thoughts?